docs/build-on-linea/tooling/data-indexers/Tracking_Full_NFT_data_on_Linea_Using_NFTScan_API.md

```diff
@@ -0,0 +1,101 @@
+---
```

Comment:
Can you rename this file to nft-scan
Failing with this error

`File path must be only lowercase letters, digits, - (dash) and _ (underscore)`

Also the heading to NFTScan so that the side bar doesn't show up as a full sentence

Feel free to also split this up into two files! This file being just a more generic introduction, and then you can add a post under the learn > community tutorials to go more in depth into how to use NFTScan

Happy to share community tutorials when they come through